sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -517,6 +517,16 @@ public Node getNode() {
         }
 
         public ClientCall<ProtoRequestT, ResponseT> createCall() {
+            // SECTION 1
+            System.out.format("Node: %s", node.address != null ? node.address.getAddress().toString() : "NULL");
```

Comment:
You should be using `logger.trace` instead of `System.out.format`

Reply:
Right below the System.out lines is the logger.trace. I intend to remove the System.out lines once I can get logging on my system working.